### PR TITLE
XWIKI-17915: Message stream messages are not displayed correctly in the rss feeds

### PR DESCRIPTION
**Pull request by manuelleduc**
https://jira.xwiki.org/browse/XWIKI-17915

- Defines dedicated templates to display the message in RSS feeds
- Defines dedicated titles for the event, used as the titles of the message in the RSS feeds
- Hides the changes when no version is provided

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-notifications/src/main/resources/templates/notification/rss/mentions.mention.vm [hunk 1]
```diff
@@ -56,8 +56,11 @@
             #set ($lastIndex = $event.events.size() - 1)
             #set ($origdoc = $xwiki.getDocument($event.document, $event.events[$lastIndex].documentVersion).previousDocument)
         #end
-        #set ($externalURL = $newdoc.getExternalURL('view', "viewer=changes&rev1=${origdoc.version}&rev2=${newdoc.version}"))
-        #set ($diffLink = "<a href='$externalURL'>${newdoc.version}</a>")
-        $services.localization.render('notifications.rss.seeChanges', 'xwiki/2.1', [$diffLink])
+        #if ($newdoc.version)
```
Comment (surli) [inline xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-notifications/src/main/resources/templates/notification/rss/mentions.mention.vm:59]: Those changes are not related to message stream: should be part of another commit I guess.

**Review by surli — CHANGES_REQUESTED** [review]

File: xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-notifications/src/main/resources/templates/notification/rss/mentions.mention.vm (hunk 1 above)
Comment (manuelleduc) [reply xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-notifications/src/main/resources/templates/notification/rss/mentions.mention.vm]: Changes removed and moved to https://jira.xwiki.org/browse/XWIKI-17923 and https://jira.xwiki.org/browse/XWIKI-17924

**Review by surli — APPROVED** [review]
